### ISSUE TITLE
e2e tests should deploy to a separate namespace

### ISSUE THREAD

**Issue by glindste** [GoogleCloudPlatform/k8s-multicluster-ingress#111]
To make it easier to do cleanup of a botched run and have separation from existing deployments the e2e tests should deploy to their own namespace, or possibly allow for configuring which namespace to use.

**Comment by nikhiljindal** [comment]
Yes. there is an open TODO for it: https://github.com/GoogleCloudPlatform/k8s-multicluster-ingress/blob/22c516ef9fb39c64f0032de51fa35a14fdf1ce5c/test/e2e/cases/basic.go#L61
